React-Native/TodoList/src/app/screens/Books/index.js

```diff
@@ -1,12 +1,112 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { View, Text } from 'react-native';
 
-const Books = function () {
-  return (
-    <View>
-      <Text>{/* TODO: Make a book list */}</Text>
-    </View>
-  );
+import List from './components/List';
+import styles from './styles';
+
+class Books extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      books: [
```

Comment:
nono.. this is not the way. Instead of this, add a file for all this and call it. You can use ` const XXX = {...`. Also, i don't see the need in having it in the state.

Reply:
with it i had problems trying to decode the json object from the books.json file, so i decided to make a plain object... but it's true that i need to make it in another file

Reply:
oh, it is static info, i don't know why i put that in the state... big mistake